contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
```

Comment:
I've just moved the bot library (`.py` file) to the `lib` folder, I think you can leave everything else in the separate folder - this is something we'll be probably addressing soon, as we're getting more and more bots ;)